src/main/resources/templates/index.html

```diff
@@ -3,59 +3,18 @@
 <head>
     <meta charset="UTF-8">
     <title>Market Data</title>
-    <style>
-        body{font-family: Consolas,serif;}
-        .panel{ width: 400px; height: 400px; border: solid black 1px}
-    </style>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
 </head>
+<body>
 <h1>Market Data API</h1>
-<h2 id="symbol"></h2>
-<body >
+<h2 id="symbol"> </h2> <!-- I had a misunderstanding about this. I see that your JS is using innerHTML to work with this element. This is fine. It was throwing an error before because your JavaScript was not being called upon as it was located in the wrong place.-->
+<img src="placeholder.png" alt ="Placeholder image of a chart"> <!--If you want to add an image, do it like this using an alt tag. Using empty headers is bad for accessibility.-->
```

Comment:
I also added an image placeholder as a logo. I think the site could benefit from a small logo like this.